tox.ini

```diff
@@ -13,6 +13,7 @@ extend_skip =
 max-line-length = 110
 extend-exclude =
     .venv,
+    venv,
     src/dbtools-mcp-server,
     src/mysql-mcp-server,
     src/oci-pricing-mcp-server
```

Comment:
Do we need to exclude the new "oracle-db-doc-mcp-server" too? Maybe refresh the PR and make sure everything is passing the 70% or adjust the exclusion?